eclipse-collections-code-generator/src/main/resources/test/list/immutable/abstractImmutablePrimitiveListTestCase.stg

```diff
@@ -80,16 +80,16 @@ public abstract class AbstractImmutable<name>ListTestCase extends AbstractImmuta
         }
     }
 
-    @Test(expected = IndexOutOfBoundsException.class)
+    @Test
     public void get_throws_index_greater_than_size()
     {
-        this.classUnderTest().get(this.classUnderTest().size());
+        Assert.assertThrows(IndexOutOfBoundsException.class, () -> this.classUnderTest().get(this.classUnderTest().size()));
```

Comment:
Do you not want to make this a static import since you are changing this anyway?

Reply:
Oh sure, it was like that in the original but I'll change it.